src/Parser.php

```diff
@@ -0,0 +1,143 @@
+<?php
+namespace DemandwareXml;
+
+use \Exception;
```

Comment:
Either throw the `Demandware\XmlException` or maybe add a new `Demandware\ParserException`?